tenpy/algorithms/tdvp.py

```diff
@@ -116,7 +156,7 @@ def evolve(self, N_steps, dt):
         N_steps : int
             The number of steps to evolve.
         """
-        consistency_check(dt, self.options, 'max_dt', 1.,
+        consistency_check(dt, self.options, 'max_dt', 2.,
```

Comment:
Jutho's original paper (https://arxiv.org/abs/1103.0936) states that TDVP has no Trotter error, but I believe this is the original formulation that does global updates rather than the more stable local update, sweeping method he introduced later (https://arxiv.org/abs/1408.5056). So the L->R->L sweeping method still has Trotter error even though one technically evolves by the full Hamiltonian (just projected into the one- or two-site variational space)?

Reply:
Is there a reason then that we don't implement a 4th order TDVP timestep that is directly analogous to a 4th order imaginary TEBD sweep?

Reply:
I was under the impression that if the bond dimension is large enough so that the projection of the Hamiltonian is accurate, then there is no discretization / Trotter error in TDVP, so I must be wrong.